docs/SalesforceStreaming-streamingsource.md

```diff
@@ -0,0 +1,87 @@
+
+Configuration
+-------------
+
+**Reference Name:** Name used to uniquely identify this source for lineage, annotating metadata, etc.
+
+**Username:** Salesforce username.
+
+**Password:** Saleforce password.
+
+**Client Id:** Application Client Id. This is also called "Consumer Key" in Salesforce UI.
+To create Client Id user needs to create a connected application in Salesforce first.
+
+**Client Secret:** Application Client Secret. This is also called "Consumer Secret" in Salesforce UI.
+To create Client Secret user needs to create a connected application in Salesforce first.
+
+**Login Url:** Salesforce OAuth2 login url.
+
+**Topic Name:** Salesforce push topic name. Plugin will track updates from this topic. If topic does
+not exist plugin creates it. To manually create pushTopic use Salesforce workbench or
+Apex code or API.
+
+
+**Query:** Salesforce push topic query. The query is used by Salesforce to send updates to push topic.
+This field not required, if you are using an existing push topic.
+
+
+**Notify On Create:** Push topic property, which specifies if a create operation should generate a record.
+
+
+**Notify On Update:** Push topic property, which specifies if a update operation should generate a record.
+
+
+**Notify On Delete:** Push topic property, which specifies if an delete operation should generate a record.
+
+
+**Notify For Fields:** Push topic property, which specifies how the record is evaluated against the
+PushTopic query The NotifyForFields values are:<br>
+_All_	- Notifications are generated for all record field changes, provided the evaluated records match
+the criteria specified in the WHERE clause.<br>
+_Referenced (default)_ -	Changes to fields referenced in the SELECT and WHERE clauses are evaluated.
+Notifications are generated for the evaluated records only if they match the criteria specified
+in the WHERE clause.<br>
+_Select_	- Changes to fields referenced in the SELECT clause are evaluated. Notifications are generated
+for the evaluated records only if they match the criteria specified in the WHERE clause.<br>
+_Where_	- Changes to fields referenced in the WHERE clause are evaluated. Notifications are generated
+for the evaluated records only if they match the criteria specified in the WHERE clause.
+
+**SObject Name:** Salesforce object name to read. If value is provided, plugin will get all fields for this object from
+Salesforce and generate SOQL query (`select <FIELD_1, FIELD_2, ..., FIELD_N> from ${sObjectName}`).
+Ignored if SOQL query is provided.
+
+**Error Handling:** Strategy used to handle erroneous records. Acceptable values are Skip on error, Stop on error.<br>
+Skip on error - Ignores erroneous records.<br>
+Stop on error - Fails pipeline due to erroneous record
+
+**Schema:** The schema of output objects.
+The Salesforce types will be automatically mapped to schema types as shown below:
+
+
+| Schema type  | Salesforce type | Notes |
```

Comment:
Please do not use md table, on CDAP Web UI its output is incorrect, use table in a form of ASCII.